modules/data.atmosphere/R/met.process.R

```diff
@@ -12,21 +12,14 @@
 ##' @param dir  directory to write outputs to
 ##' 
 ##' @author Elizabeth Cowdery, Michael Dietze
-met.process <- function(site, input_met, start_date, end_date, model, host, bety, dir, browndog=NULL){
+met.process <- function(site, input_met, start_date, end_date, model, host, dbparms, dir, browndog=NULL){
   
   require(RPostgreSQL)
```

Comment:
This is not needed, this will be done in db.open and will use the driver in dbparms. This way in theory we can use different databases.